root/static/scripts/edit/URLCleanup.js

```diff
@@ -1137,7 +1137,7 @@ const CLEANUPS = {
             if (prefix === 'master') {
               return {
                 error: l(
-                  `Discogs master links group several releases,
+                  `Discogs “master” links group several releases,
```

Comment:
I'd prefer `“/master”` to make it clearer.

Reply:
It was quoting the content of the Discogs page. To refer to an URL part, I would probably use `<code>` instead:
* either `Discogs <code>/master</code> links`
* or `Discogs links with <code>/master</code> in path`

or any better phrased variant… :)

Reply:
I like the first :)

Reply:
Fixed at https://github.com/metabrainz/musicbrainz-server/pull/1536/commits/2683b3e6da835c600ffb2dabda28a13eb0853977#diff-87814c26df342be7e4c2a85531bbd47cL1140-R1153